src/test/tests.cpp

```diff
@@ -428,6 +428,39 @@ TEST_CASE("replaceAllOf") {
 	}
 }
 
+TEST_CASE("cleanForMaya") {
+	SECTION("empty") {
+		std::wstring testString = L"";
```

Comment:
initializing to empty string is not necessary

Reply:
removed (also in other cases above)